exercises/practice/protein-translation/src/test/java/ProteinTranslatorTest.java

```diff
@@ -149,7 +150,54 @@ public void testTranslationStopsIfStopCodonInMiddle1() {
     @Test
     public void testTranslationStopsIfStopCodonInMiddle2() {
         assertThat(proteinTranslator.translate("UGGUGUUAUUAAUGGUUU"))
-            .containsExactly("Tryptophan", "Cysteine", "Tyrosine");
+                .containsExactly("Tryptophan", "Cysteine", "Tyrosine");
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void testEmptyRnaSequenceResultInNoproteins() {
+        assertThat(proteinTranslator.translate("")).isEmpty();
+    }
```

Comment:
Note: The order of the tests in the canonical data may be ordered like that on purpose, for instance to cover basic test cases before more elaborate ones. This should aid students in their implementation.

To that end, we try to keep the order of the tests in the Java track in the same way as they are ordered in the canonical data. That would mean that this test for an empty sequence becomes the first test. Note that this also means that it shouldn't have an `@Ignore` tag and that the existing `testTranslationOfRnaToProteinList` method should be `@Ignore`d.

Reply:
Oh okey good to know! I updated the order to match the canonical data!